mkosi/config.py

```diff
@@ -2160,7 +2159,7 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         yield section, "", ""
 
 
-PE_ADDON_SETTINGS = (
+PE_ADDON_SETTINGS: tuple[ConfigSetting[Any], ...] = (
```

Comment:
I don't think we win anything by this being tuples, so if we make these lists, the typing becomes a nicer
 ```suggestion
PE_ADDON_SETTINGS: list[ConfigSetting[Any]] = [
```